packages/wonder-blocks-dropdown/components/option-item.js

```diff
@@ -6,16 +6,18 @@ import PropTypes from "prop-types";
 
 import Color, {mix, fade} from "@khanacademy/wonder-blocks-color";
 import Spacing from "@khanacademy/wonder-blocks-spacing";
-import {LabelLarge} from "@khanacademy/wonder-blocks-typography";
+import {LabelMedium} from "@khanacademy/wonder-blocks-typography";
 import {View, getClickableBehavior} from "@khanacademy/wonder-blocks-core";
 
-import type {StyleType} from "@khanacademy/wonder-blocks-core";
+import type {AriaProps, StyleType} from "@khanacademy/wonder-blocks-core";
 
 import {DROPDOWN_ITEM_HEIGHT} from "../util/constants.js";
 import Check from "./check.js";
 import Checkbox from "./checkbox.js";
 
 type OptionProps = {|
+    ...AriaProps,
```

Comment:
If we accept all `AriaProps` we should be passing all them to the underlying `View`, not just `aria-label`.